src/binaryapi/ActiveSymbols.js

```diff
@@ -78,9 +79,9 @@ export default class ActiveSymbols {
 
         // Categorize symbols in order defined by another array; there's probably a more
         // efficient algo for this, but for just ~100 items it's not worth the effort
-        const order = ['forex', 'indices', 'stocks', 'commodities', 'synthetic_index'];
+        // const order = ['forex', 'indices', 'stocks', 'commodities', 'synthetic_index'];
```

Comment:
can we remove this line?

Reply:
@balakrishna-binary  that's my bad. just correct it.